app/controllers/application_controller.rb

```diff
@@ -17,10 +17,17 @@ class ApplicationController < ActionController::Base
   before_action :current_organization
   before_action :organization_redirect, if: -> { params["current-organization"] }
   before_action -> { Current.user_interface = current_user&.persisted? }
+  after_action -> { set_last_controller_name }
   impersonates :user
 
   protected
 
+  def set_last_controller_name
+    unless controller_name.include? "sparkline"
+      session[:last_controller_name] = controller_name
+    end
+  end
+
```

Comment:
Are we still using `last_controller_name` anywhere?

Reply:
good catch